modules/ROOT/pages/index.adoc

```diff
@@ -2,14 +2,40 @@
 
-* RESTful API Modeling Language (RAML) 1.0 and 0.8 versions
-* Web Service Description Language (WSDL)
+* xref:apikit-4-for-graphql.adoc[] supports GraphQL APIs. 
+* xref:apikit-4-for-odatav2.adoc[] and xref:apikit-4-for-odatav4.adoc[] support OData APIs. 
+* xref:apikit-4-for-rest.adoc[] supports RAML and OAS APIs.
+* xref:apikit-4-for-soap.adoc[] supports SOAP APIs. 
 
-APIkit simplifies the design and implementation of REST APIs. APIkit generates minimal Mule flows based on either a RAML or WSDL that meet the communication operations required by a REST or web service contract, respectively.
+With APIkit, incoming requests are intelligently routed to the appropriate endpoints based on the defined API contracts. This routing mechanism simplifies request handling, improves performance, and ensures that requests are processed by the correct API resources.
 
-MuleSoft recommends Design Center for creating and synching RAML code; however, Studio includes a RAML editor that provides syntax validation and suggestions if you want to perform the entire development within Studio. For generating flows using APIkit, use Studio to automate the flow generation process, or you can create APIkit flows for Mule in an XML editor.
+Although each module is designed for a family of web services, all modules share common usage patterns. So if you are a Mule developer that works with SOAP APIs, you can easily get used to working with GraphQL, RAML, OData, or OAS APIs.
 
-APIkit projects can run onsite or on CloudHub. After generating the API, you can simulate calls using API Console.
\ No newline at end of file
+== Features
+
+* APIkit generates the structure of a Mule application based on a supported API specification.
+* APIkit includes sources and operations that perform complex input validations according to the requirements described in the API specification.
+* APIkit for REST module includes an API console that enables you to experiment with the API project as you develop it.
+* Some APIkit modules include extensions for xref:studio::index.adoc[Anypoint Studio] and xref:anypoint-code-builder::index.adoc[Anypoint Code Builder] for enhancing your editing experience if you choose to use the supported development environments.
+
+== Prerequisites
+
+You need to have access to xref:studio::index.adoc[Anypoint Studio] or xref:anypoint-code-builder::index.adoc[Anypoint Code Builder] to scaffold a supported API specification with APIkit.
+
+== APIkit Versions
+
+APIkit is designed to work as a module with Mule runtime version 4. Documentation for the Mule 3 implementation of APIkit is available https://docs.mulesoft.com/apikit/3.x/[here]. If you use APIkit on Mule 3, the module version matches the runtime version.
+
+By default, Studio includes APIkit for REST and APIkit for SOAP modules. If you need GraphQL or OData modules, follow the installation instructions in each section.
```

Comment:
```suggestion
By default, Studio includes the APIkit for REST and APIkit for SOAP modules. If you need GraphQL or OData modules, follow the installation instructions in each section.
```

Reply:
I tried to be consistent throughout the articles giving the modules always the same name.
There are hundreds of "GraphQL modules" but only one "APIkit for GraphQL module" if you make a Google search.